### PR TITLE
extra-kotlin

### PR DESCRIPTION
**Pull request by kashike**
Replaces #139.

### CODE REVIEW

**Review by BomBardyGamer — COMMENTED** [review]
Very nice looking DSL! Would love to have this in the future!

**Review by BomBardyGamer — APPROVED** [review]

**Comment by CrushedPixel** [comment]
Beautiful DSL, indeed!

File: extension-kotlin/build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
+plugins {
+  id "org.jetbrains.kotlin.jvm" version "1.4.30-M1"
```
Comment (zml2008) [inline extension-kotlin/build.gradle:2]:
```suggestion
  id "org.jetbrains.kotlin.jvm" version "1.4.30"
```

**Review by zml2008 — CHANGES_REQUESTED** [review]

**Review by zml2008 — APPROVED** [review]

**Review by zml2008 — APPROVED** [review]

**Comment by kashike** [comment]
@ItsDoot could you please review this?

File: extra-kotlin/build.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
+plugins {
+  id "org.jetbrains.kotlin.jvm" version "1.4.30-M1"
```
Comment (ItsDoot) [inline extra-kotlin/build.gradle]: 1.4.31

**Review by ItsDoot — APPROVED** [review]

**Pull request merged**